site/smart-accounts/accounts/light-account.md

```diff
@@ -14,35 +14,49 @@ head:
 
 # Light Account
 
-Light Account is a secure, gas-optimized, [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart account implementation.
+### Secure, optimized, and extendable
 
-We started with the Ethereum Foundation’s canonical [SimpleAccount](https://github.com/eth-infinitism/account-abstraction/blob/develop/contracts/samples/SimpleAccount.sol) and added key improvements for production app developers:
+In Web3, efficiency, trust, and security are non-negotiable, especially with your account. That's why we built Light Account! With Light Account, we’ve turbocharged Ethereum smart accounts, reducing gas costs and supporting ERC-1271 signatures to add portability amongst all dApps. Plus, we've got your back with ownership transfer, ensuring you're never locked into a single signer. Our benchmark results speak for themselves, giving you the confidence to use a fully audited, optimized, and [open-sourced](https://github.com/alchemyplatform/light-account) smart account solution.
 
-- significantly reduced gas costs
-- [ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) signature support to ensure users can sign messages, such as on OpenSea
-- ownership transfer so that users won’t get locked into a single `Signer`
+### Future-Proofed with Modular 6900 Accounts:
+
+We also built Light Account with the future in mind. Light Account is forward-compatible and upgradeable for future, more optimized versions and the upcoming modular 6900 account standard, ensuring you'll stay on the cutting edge of EVM accounts. 
+
+### The Responsible Choice
+
+At Alchemy, in partnership with the community, we focus on the fundamentals and build our solutions from the ground up. Your trust and safety are paramount in the new digital frontier, so ensure you and your users are backed by a thoughtful craft and standard. The future of accounts is within the Alchemy Family of Smart Accounts - Optimized, Open, and Secure.
 
 ## Using Light Account
```

Comment:
in my opinion this doc is getting long (which is good!) and for clarity we should split them up into two separate docs:
* Why Light Account
* Using Light Account

left more details in the [asana ticket](https://app.asana.com/0/1205598834384806/1205608436455254/f).  This doesn't have to be a blocker for this PR, we can do it later if easier